pages/common/pio-project.md

```diff
@@ -0,0 +1,20 @@
+# pio project
+
+> Tool to manage PlatformIO projects.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/project/index.html>.
+
+- Initialize a new PlatformIO based project (defaults to current directory if `--project-dir` is omitted):
+
+`pio project init --project-dir {{path/to/project_directory}}`
+
+- Initialize a new PlatformIO based project, specifying a board ID and several project options:
```

Comment:
This is a pretty complicated example. Can we simplify it at all?

Remember the goal in tldr pages isn't to document _everything_ - just the most common options that you'll need.